src/test/testdata/expected/lsl/targetAnnotation.lsl

```diff
@@ -1,12 +1,12 @@
 libsl "1.0.0";
 library targetAnnotation;
-
 types {
     Int(int32);
     A(int16);
 }
+annotation Target;
 automaton B : Int {
 }
 automaton A : Int {
-    fun foo(@target self: B);
-}
+    fun foo(@Target self: Int);
```

Comment:
Okay, again. I think the parser shouldn't modify the representation this way. The expected outputting specification should be pretty the same as inputting one

Reply:
Returned functionality which prints target automaton as type